studioShared/lib/queries/customerCases.ts

```diff
@@ -36,7 +36,9 @@ export const CUSTOMER_CASE_QUERY = groq`
       "name": ${translatedFieldFragment("name")},
       "duration": ${translatedFieldFragment("duration")},
       "sector": ${translatedFieldFragment("sector")},
-      "delivery": ${translatedFieldFragment("delivery")},
+      "deliveries": delivery[] {
+        "delivery": ${translatedFieldFragment("delivery")},
+      },
```

Comment:
We should be consistent with "deliveries"/"delivery" in schema and GROQ response. I vote for "deliveries" everywhere for the array, and "delivery" for the string value of each array item.